nixos/modules/security/hidepid.nix

```diff
@@ -36,7 +36,12 @@ in
   config = mkIf cfg.enable {
     systemd.services.hidepid = {
       wantedBy = [ "sysinit.target" ];
+      # Conceptually, it'd make sense to order this after systemd-remount-fs,
+      # but we cannot be sure that systemd-remount-fs will run; ordering this
+      # service after local-fs.target will work in either case
+      # (systemd-remount-fs is ordered before local-fs, by default).
       after = [ "local-fs.target" ];
```

Comment:
Wny not add `systemd-remount-fs.service` to `after` then? It would order them properly if `systemd-remount-fs` runs without requiring it to run it all.